lib/template.html

```diff
@@ -131,18 +151,27 @@
   <meta property="og:site_name" content="{{ seo_site_title }}" />
 {% endif %}
 
-{% if seo_page_image %}
-<meta property="og:image" content="{{ seo_page_image }}" />
-  {% if page.image.height %}
-    <meta property="og:image:height" content="{{ page.image.height }}" />
+{% if page.image %}
+  {% if page.image.facebook %}
+    {% assign local_seo_page_image = seo_page_image_facebook %}
+    {% assign local_seo_page_image_height = seo_page_image_facebook_height %}
+    {% assign local_seo_page_image_width = seo_page_image_facebook_width %}
+  {% else %}
+    {% assign local_seo_page_image = seo_page_image_default %}
```

Comment:
Good points, all. I'm a :shipit: if we have the tests updated. Don't want to beat a dead horse.